### PR TITLE
2 court info

### PR DESCRIPTION
**Pull request by davclark**
Implements a simple widget to display the selected court info (court name passed in via props). Data is currently defined as a variable inside the component.

### CODE REVIEW

**Comment by davclark** [comment]
@kg6zvp did you want to review this properly? It's not complex - so I can also just merge without review.

File: src/Court.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  yuma:      {url: "http://www.yumacountyaz.gov/government/courts/superior-court",
+              fullname: 'Yuma County Superior Court',
+              phone_info: "Clerk's Office (928) 817-4222"},
+}
+
+function Court(props) {
+  const selected_court = court_info[props.court]
+  return (
+    <div className="Court">
+      <p><a href={selected_court.url}>{selected_court.fullname}</a> – {selected_court.phone_info}</p>
```
Comment (kg6zvp) [inline src/Court.js]: Could you make the phone number a `tel:` link? Otherwise, looks good to me. Merge whenever. 👍

Comment (davclark) [reply src/Court.js]: That's not completely trivial given React's escaping rules and the frequent presence of more than 1 phone number. I'll merge for now and add an issue.

**Comment by davclark** [comment]
Opened an issue about `tel:` links in #23

**Pull request merged**